server/api.go

```diff
@@ -472,6 +472,8 @@ func (a *API) oauthRedirectHandler(w http.ResponseWriter, r *http.Request) {
 
 	_, _ = w.Write([]byte(fmt.Sprintf("<html><body><h1>%s</h1><p>You can close this window.</p></body></html>", connectionMessage)))
 
+	_, _ = a.p.updateAutomutingOnUserConnect(mmUserID)
```

Comment:
Can we log `LogWarn` any error that's returned here to avoid burying it?

Reply:
This code is actually a bit misleading since we do log any errors inside `updateAutomutingOnUserConnect`. I wanted the method to log its own errors to keep the automute logic more self-contained, but I still wanted the method to return errors because that makes testing easier